membership-attribute-service/app/configuration/Config.scala

```diff
@@ -52,4 +52,9 @@ object Config {
     val enabled = Try{config.getBoolean("logstash.enabled")}.toOption.contains(true)
   }
 
+  object Mobile {
+    val subscriptionURL = config.getConfig("mobile.subscription.base")
```

Comment:
I mentioned already on chat, but any piece of config is a risk to prod as it is only checked at run time.  If the config is not a secret, then there's no reason not to put it in the source code as a pattern match or if statement.  This gets full static checking and can be reviewed and rolled back together with the changes.
The disadvantage is you have to know all your environments at compile time, but a PR to add an environment wouldn't be arduous for us.

Reply:
So just to confirm, you're okay with me breaking the existing pattern and have a if "PROD" in here?

Reply:
tl-dr: yes I'm ok with that. I think compile time safety > following a pattern

Personally I wouldn't even put it on that line, but at the relevant part of the code, but I appreciate a big refactor isn't what you want at this point.
For example the stripe base url is a default parameter https://github.com/guardian/support-frontend/blob/c47c47b2726a12f586b062738c97a405b0377b97/support-workers/src/main/scala/com/gu/stripe/StripeService.scala#L15